test/parallel/test-http-client-aborted-event.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+require('../common');
+var http = require('http');
+var assert = require('assert');
+
+var server = http.Server(function(req, res) {
```

Comment:
It’s a good idea to wrap all callbacks in `common.mustCall(…)` to make sure they are actually invoked during the test